lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -765,26 +765,26 @@ public function getModExpression($expression1, $expression2)
     /**
      * Returns the SQL snippet to trim a string.
      *
-     * @param string         $str  The expression to apply the trim to.
-     * @param integer        $pos  The position of the trim (leading/trailing/both).
-     * @param string|boolean $char The char to trim, has to be quoted already. Defaults to space.
+     * @param string      $str  The expression to apply the trim to.
+     * @param int         $pos  The position of the trim (leading/trailing/both).
```

Comment:
Should `$pos` be renamed to `$mode`?

Reply:
Probably, will change that - didn't check variable names to be honest.